packages/web-components/fast-components/test/.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
+public
```

Comment:
should we have a separate `.gitignore` in this sub folder or should we add it to the `.gitignore` in `fast-components/`?

Reply:
the default .gitignore should be good enough